packages/vscode-extension/lib/expo_router_plugin.js

```diff
@@ -17,9 +17,12 @@ function useRouterPluginMainHook({ onNavigationChange }) {
   const pathname = routeInfo?.pathname;
   const params = routeInfo?.params;
 
+  const displayParams = new URLSearchParams(params).toString();
```

Comment:
info: copy pasted since as I know the v2 plugin will be removed and similar approach was recommended in previous implementation